packages/alba/alba.0.4.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "alba"
+synopsis: "Alba compiler"
+maintainer: "Helmut Brandl <helmut.brandl@gmx.net>"
+authors: "Helmut Brandl <helmut.brandl@gmx.net>"
+homepage: "http://albatross-lang.sourceforge.net"
+license: "GPLv2"
+dev-repo: "git+https://github.com/hbr/albatross.git"
+bug-reports: "http://github.com/hbr/albatross"
+build: [
+  [make "test_alba"]
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@runtest" ]
+  ["dune" "build" "@doc" "-p" name]
+  ["sh" "./.copy_docu.sh"]
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "linenoise"
+  "ocamlfind" {build}
+  "menhir" {build}
+  "dune" {build}
+  "odoc" {build}
+  "js_of_ocaml" {build}
+  "js_of_ocaml-ppx" {build}
```

Comment:
```suggestion
  "js_of_ocaml-ppx"
```
This is usually appearing without the build tag, I guess it may have a runtime. Or you just need it to generate executables?